build/.gitignore

```diff
@@ -0,0 +1 @@
+*.pdf
```

Comment:
`*.pdf` files are ignored by `.gitignore`, but there are 3 `.pdf` files are included in the repo now. We should remove them:
- sections/graphviz-images/graphviz-dbft-sm.pdf
- sections/graphviz-images/graphviz-dbft-v2-recover.pdf
- sections/graphviz-images/graphviz-pbft.pdf

Reply:
@igormcoelho
What about these files? Better to ignore everything, right?

Reply:
Done.